concepts/05 Widgets/Lookup/03 Data Binding/05 Simple Array/05 Array Only.md

```diff
@@ -129,12 +242,84 @@ You can create a [Query](/concepts/70%20Data%20Binding/5%20Data%20Layer/6%20Quer
         // ...
     })
 
-    <!--HTML-->
-    <dx-lookup
-        [dataSource]="getSortedProducts()"
-        valueExpr="price"
-        displayExpr="name">
-    </dx-lookup>
+##### Vue
+
+    <template>
+        <DxLookup
+            :data-source="getSortedProducts()"
+            value-expr="price"
+            display-expr="name"
+        />
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { DxLookup } from 'devextreme-vue/lookup';
+    import query from "devextreme/data/query";
+
+    export default {
+        components: {
+            DxLookup
+        },
+        data() {
+            return {
+                products: [
+                    { name: 'HD Video Player', price: 100 },
+                    // ...
+                ]
+            };
+        },
+        methods: {
+            getSortedProducts () {
+                return query(this.products)
+                    .sortBy("price", true)
+                    .toArray();
+            }
+        }
+    }
+    </script>
+
+##### React
+
+    import React from 'react';
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { Lookup } from 'devextreme-react/lookup';
+    import query from "devextreme/data/query";
+
+    class App extends React.Component {
+        constructor(props) {
+            super(props);
+
+            this.products = [
+                { name: 'HD Video Player', price: 100 },
+                // ...
+            ];
+
+            this.getSortedProducts = this.getSortedProducts.bind(this);
+        }
+
+        getSortedProducts() {
+            return query(this.products)
+                .sortBy("price", true)
+                .toArray();
+        }
+
+        render() {
+            return (
+                <Lookup
+                    dataSource={this.getSortedProducts()}
```

Comment:
Shouldn't it be dataSource={this.getSortedProducts}?

Am I missing something?

Reply:
No, data source is the result of `getSortedProducts` function call